main.py

```diff
@@ -195,155 +240,107 @@ async def give_random(channel, number_of_issues):
+    if isinstance(error, discord.ext.commands.errors.BadArgument) or isinstance(error, discord.ext.commands.errors.MissingRequiredArgument):
+        await ctx.channel.send(embed=reply)
 
-    if command == 'what':
-        reply.title = "Command list for GitHub Helper"
-        reply.description = '''`!rand <N>`
-Retrieves N random open, non-stale issues
-
-`!review <N>`
-Retrieves N least-recently-updated PR awaiting your review
-
-`!q <N>`
-Retrieves N least-recently-updated PR in the review queue'''
-        await channel.send(embed=reply)
-        return
-
-    elif command == "rand":
-        if len(args) != 1:
-            await channel.send(embed=reply)
-            return
-        try:
-            number_of_issues = int(args[0].strip())
-            if number_of_issues < 1:
-                raise ValueError("Positive integer needed")
-        except ValueError:
-            reply.description = "I can't give you un-natural issues." + \
-                " I'm not a monster!!"
-            await channel.send(embed=reply)
-            return
-        if number_of_issues > 10:
-            number_of_issues = 10
-            reply.description = "Let's curb that enthusiasm.. just a little"
-            await channel.send(embed=reply)
-        await give_random(channel, number_of_issues)
-        return
 
-    elif command == "q" or command == "review":
-        if len(args) != 1:
-            await channel.send(embed=reply)
-            return
-        try:
-            number_of_issues = int(args[0].strip())
-            if number_of_issues < 1:
-                raise ValueError("Positive integer needed")
-        except ValueError:
-            await channel.send("This queue is 100% natural. Check your orders")
-            return
-        if number_of_issues > 10:
-            number_of_issues = 10
-            reply.description = "Let's curb that enthusiasm.. just a little"
-            await channel.send(embed=reply)
-
-        author = message.author
-        if command == "q":
-            author = None
-        elif isinstance(author, discord.Member):
-            author = author.nick or author.name
-        elif isinstance(author, discord.User):
-            author = author.name
-
-        await review_q(channel, number_of_issues, author)
-        return
-    return
-
-
-def read_config(filename):
-    with open(filename, "r") as f:
-        return json.load(f)
+@bot.command(name="review")
+async def haha(ctx):
+    reply = discord.Embed()
+    reply.set_image(url="https://media.giphy.com/media/kegHkRsheJk3fjOg3D/giphy.gif")
+    await ctx.channel.send(embed=reply)
 
 
 async def oneshot(channel_id, n):
-    await client.wait_until_ready()
-    await give_random(client.get_channel(channel_id), n)
-    await client.close()
+    await bot.wait_until_ready()
+    await command_function(ctx=None, number_of_issues=n, noun="issue", channel=bot.get_channel(channel_id), name="rand", conf=read_config("command_config.json")["rand"])
```

Comment:
The filename should appear once only. Also, wouldn't the global variable be read by the time interpreter comes down till here?

Reply:
It will be. Removed.